Card/src/test/java/com/braintreepayments/api/CardClientUnitTest.java

```diff
@@ -36,7 +36,7 @@
     @Before
     public void beforeEach() throws JSONException {
         context = mock(Context.class);
-        card = mock(Card.class);
+        card = new Card();
```

Comment:
👀  Just cleanup while we're here?

Reply:
No this change resolved some test failures related to not mocking all the underlying setters on card. Figured just using an actual Card object made more sense then mocking everything out.